### ISSUE TITLE
Feedback on configure-aws-ecs-with-terraform.md

### ISSUE THREAD

**Issue by adamlogic** [JeffMorhous/judoscale-content#1]
Great article! I have no major concerns, just a few small bits of feedback...

> Terraform gives you a confirmation of the success, but you're probably wondering if things are actually happening in AWS. At this point, it's easy to check!

Love this part! For me this really helps to understand what Terraform is doing.

> An ECS cluster is not very useful on its own. We'll use Terraform to create an ECS task definition that defines our application.

Might be useful to very briefly describe the relationship between clusters, services, and tasks. A diagram would probably be helpful for that. Not sure if you've used [Excalidraw](https://excalidraw.com), but we pretty much use it exclusively for making diagrams in our posts.

> You may wish to use an NGINX sidecar with your application for several reasons, such as using [an autoscaler like Judoscale](https://judoscale.com/docs/aws-getting-started).

I think I would discuss autoscaling before NGINX. It feels unclear at this point why NGINX is useful, and the reader doesn't yet have context for the relationship between Judoscale and NGINX.

If you talk about autoscaling and request queue time first, you can have a nice segue into why we need to include an NGINX sidecar.

**Comment by JeffMorhous** [comment]
Thanks a bunch for this feedback @adamlogic ! I just pushed a commit that I think addresses these - https://github.com/JeffMorhous/judoscale-content/commit/5f79b701085a20f2d82a1fd5f8dfdbc6560344d6

I hadn't used Excalidraw before, that's awesome! I made a diagram that follows a similar style to those in [this existing Judoscale blog post](https://judoscale.com/blog/how-our-ecs-autoscaling-works). Let me know what you think!
